tests/test_create.py

```diff
@@ -1205,13 +1202,6 @@ def test_install_update_deps_only_deps_flags(self):
             assert package_is_installed(prefix, "flask=0.12.2")
             assert package_is_installed(prefix, "jinja2>2.9")
 
-    @pytest.mark.skipif(on_win, reason="tensorflow package used in test not available on Windows")
```

Comment:
removing this is intentional, as ``--freeze-installed`` is the default behavior now.  However, it's not completely frozen as it was for this test - only frozen as much as possible, but still within the bounds of the explicit constraints in the history.  This test no longer raises the expected UnsatisfiableError.